src/main/java/duke/command/EditTaskMarkCommand.java

```diff
@@ -0,0 +1,60 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Command to edit the task to be either marked or unmarked.
+ */
+public class EditTaskMarkCommand extends Command {
```

Comment:
Thanks for the suggestion! I named it EditTaskMarkCommand instead as it is able to mark and unmark tasks. I was worried that if I just put MarkTaskCommand, people would assume it can only mark task and not unmark it 😅